flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SynchronousSavepointLatch.java

```diff
@@ -59,61 +66,53 @@ void setCheckpointId(final long checkpointId) {
 		}
 	}
 
-	boolean blockUntilCheckpointIsAcknowledged() throws Exception {
+	void blockUntilCheckpointIsAcknowledged() throws Exception {
```

Comment:
I would suggest to change the exception types for some methods like this more precisely to `InterruptedException` because it might be easier to see for caller that there might be a special reaction to perform (e.g. restoring the interrupted status).